templates/broadband-signup/layout/index.js

```diff
@@ -1,7 +1,7 @@
 import summary from './summary.js';
+import header from './header.js';
+import footer from './footer.js';
 import notFound from '../../shared/not-found.js';
```

Comment:
not-found should probably get the same treatmen.

Reply:
I was thinking of error and not-found: both not partials of a layout, but not sections as well. Both are content for the main target (like sections), so in the end, I put error.js' to the root of each domain's:
```
domain
- error.js
- sections
- layout
```

are you ok with such treatment? can do same for not-found

Reply:
Sure, that's fine by me.

Reply:
ok, will do in this PR

Reply:
oh no, actually I'll rather open a separate PR because I've touched error.js in another branch :D